src/Jackalope/RepositoryFactoryJackrabbit.php

```diff
@@ -44,6 +44,7 @@ class RepositoryFactoryJackrabbit implements RepositoryFactoryInterface
         'jackalope.logger' => 'Psr\Log\LoggerInterface: Use the LoggingClient to wrap the default transport Client',
         Session::OPTION_AUTO_LASTMODIFIED => 'boolean: Whether to automatically update nodes having mix:lastModified. Defaults to true.',
         'jackalope.jackrabbit_force_http_version_10' => 'boolean: Force HTTP version 1.0, this can in solving problems with curl such as https://github.com/jackalope/jackalope-jackrabbit/issues/89',
+        'jackalope.jackrabbit_curl_options.*' => 'mixed: Additional global curl-options',
```

Comment:
i would prefer a single option `jackalope-jackrabbit_curl_options` that contains a hashmap. then users can also use the curl constants to define the options. and you don't need the loop to extract and separate options.